### ISSUE TITLE
JPM is required on installing Blade

### ISSUE THREAD

**Issue by ipeychev** [gamerson/lxc-client-extensions-workspace#1]
➜  ~ java -version                                                                                             
openjdk version "11.0.15" 2022-04-19 LTS
OpenJDK Runtime Environment Zulu11.56+19-CA (build 11.0.15+10-LTS)
OpenJDK 64-Bit Server VM Zulu11.56+19-CA (build 11.0.15+10-LTS, mixed mode)
➜  ~ curl https://raw.githubusercontent.com/liferay/liferay-blade-cli/master/cli/installers/global -fsSL | bash
Error
  0. [UnixException.translateToIOException] Failed java.nio.file.AccessDeniedException: /Library/Java/PackageManager
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10.8M  100 10.8M    0     0  67.2M      0 --:--:-- --:--:-- --:--:-- 68.9M
bash: line 1: /jpm: No such file or directory

**Comment by gamerson** [comment]
Fixed in #2

**Comment by gamerson** [comment]
the fix is to use `local` instead of `global` script.  So
```
curl https://raw.githubusercontent.com/liferay/liferay-blade-cli/master/cli/installers/local -fsSL | bash
```
